### PR TITLE
Replacing log calls

### PR DESCRIPTION
**Pull request by rishubhjain**
Using log function from tendrl/commons/utils/log_utils.py instead of explicit calls to Event and Message

### CODE REVIEW

**Comment by rishubhjain** [comment]
@r0h4n  yes, I tested the code using my tests, could you please paste your error so that I may make the necessary changes

**Comment by r0h4n** [comment]
@rishubhjain please install etcd and tendrl-node-agent on your machine and try the PR, just need to know if you have verified by actually running the node-agent

**Comment by rishubhjain** [comment]
@r0h4n Verified , the logger records the messages as per desired

File: tendrl/commons/__init__.py [hunk 1]
```diff
@@ -47,68 +44,37 @@ def __init__(self, ns_name="tendrl", ns_src="tendrl.commons"):
...
-                    publisher=NS.publisher_id,
-                    payload={"message": "Setup Tendrl definitions (.yml) for namespace.%s" %
-                             self.ns_name
-                             }
-                )
-            )
-        except KeyError:
-            sys.stdout.write("Setup Tendrl definitions (.yml) for namespace.%s" %
-                             self.ns_name)
+        logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: please make this a "debug" , since not intended for the users

File: tendrl/commons/__init__.py [hunk 2]
```diff
@@ -47,68 +44,37 @@ def __init__(self, ns_name="tendrl", ns_src="tendrl.commons"):
...
-                        payload={
-                            "message": "Validating registered (.py) flows in"
-                                       " %s.flows" % raw_ns
-                            }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Validating registered (.py) flows in %s."
-                                 "flows" % raw_ns)
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: please make this a "debug" too

File: tendrl/commons/__init__.py [hunk 3]
```diff
@@ -119,37 +85,18 @@ def _validate_ns_flow_definitions(self, raw_ns, defs):
...
-                        Message(
-                            priority="error",
-                            publisher=NS.publisher_id,
-                            payload={"message": msg
-                            }
-                        )
-                    )
-                except KeyError:
-                    sys.stderr.write(msg)
+                logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 4]
```diff
@@ -119,37 +85,18 @@ def _validate_ns_flow_definitions(self, raw_ns, defs):
...
-                        publisher=NS.publisher_id,
-                        payload={"message": "Validating defined (.yml) flows"
-                                            " in %s.flows" % raw_ns
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Validating defined (.yml) flows in "
-                                 "%s.flows" % raw_ns)
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 5]
```diff
@@ -181,19 +118,9 @@ def _validate_ns_obj_definitions(self, raw_ns, defs):
...
-                        publisher=NS.publisher_id,
-                        payload={"message": "Validating registered (.py) "
-                                            "objects in %s.objects" % raw_ns
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Validating registered (.py) objects in "
-                                 "%s.objects" % raw_ns)
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 6]
```diff
@@ -259,39 +167,19 @@ def _validate_ns_obj_definitions(self, raw_ns, defs):
...
-                        publisher=NS.publisher_id,
-                        payload={"message": "Validating defined (.yml) objects"
-                                            " in %s.objects" % raw_ns
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Validating defined (.yml) objects in %s."
-                                 "objects" % raw_ns)
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 7]
```diff
@@ -361,59 +229,26 @@ def _validate_ns_obj_definitions(self, raw_ns, defs):
...
-                        publisher=NS.publisher_id,
-                        payload={"message": "Setup Config for namespace.%s" %
-                                 self.ns_name
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Setup Config for namespace.%s" %
-                                 self.ns_name)
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 8]
```diff
@@ -361,59 +229,26 @@ def _validate_ns_obj_definitions(self, raw_ns, defs):
...
-                                            self.ns_name
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Setup central store clients for "
-                                 "namespace.%s" %
-                                 self.ns_name)
 
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 9]
```diff
@@ -438,40 +273,18 @@ def setup_common_objects(self):
...
-                        payload={"message": "Setup NodeContext for namespace."
-                                            "%s" % self.ns_name
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Setup NodeContext for namespace. %s" %
-                                 self.ns_name)
-
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 10]
```diff
@@ -438,40 +273,18 @@ def setup_common_objects(self):
...
-                        payload={"message": "Setup TendrlContext for namespace"
-                                            ".%s" % self.ns_name
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Setup TendrlContext for namespace.%s" %
-                                 self.ns_name)
-
+            logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: debug

File: tendrl/commons/__init__.py [hunk 11]
```diff
@@ -619,20 +432,9 @@ def get_flow_definition(self, flow_name):
...
-                    payload={"message": "Finding objects in namespace.%s."
-                                        "objects" % self.ns_name
-                             }
-                )
-            )
-        except KeyError:
-            sys.stdout.write("Finding objects in namespace.%s.objects\n" %
-                             self.ns_name)
-
+        logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: debug

File: tendrl/commons/__init__.py [hunk 12]
```diff
@@ -645,43 +447,18 @@ def _register_subclasses_to_ns(self):
...
-                                        self.ns_name, obj_name)
-                                    }
-                            )
-                        )
-                    except KeyError:
-                        sys.stdout.write("Registering object namespace.%s."
-                                         "objects.%s\n" % (self.ns_name,
-                                                         obj_name)
-                                         )
+                    logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: make this a "debug"

File: tendrl/commons/__init__.py [hunk 13]
```diff
@@ -645,43 +447,18 @@ def _register_subclasses_to_ns(self):
...
-                                               "objects.%s.atoms" %
-                                               (self.ns_name, obj_name)
-                                }
-                            )
-                        )
-                    except KeyError:
-                        sys.stdout.write("Finding atoms in namespace.%s."
-                                         "objects.%s.atoms\n" %
-                                         (self.ns_name, obj_name))
+                    logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 14]
```diff
@@ -691,55 +468,23 @@ def _register_subclasses_to_ns(self):
...
-                                    )
-                                except KeyError:
-                                    sys.stdout.write("Registering atom "
-                                                     "namespace.%s.objects.%s."
-                                                     "atoms.%s" %
-                                                     (self.ns_name,
-                                                      obj_name,
-                                                      atom_cls[1].__name__)
-                                                     )
+                                logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 15]
```diff
@@ -691,55 +468,23 @@ def _register_subclasses_to_ns(self):
...
-                                                               obj_name)
-                                         }
-                            )
-                        )
-                    except KeyError:
-                        sys.stdout.write("Finding flows in namespace.%s."
-                                         "objects.%s.flows\n" % (self.ns_name,
-                                                               obj_name))
-
+                    logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 16]
```diff
@@ -749,30 +494,11 @@ def _register_subclasses_to_ns(self):
...
-                                    )
-                                except KeyError:
-                                    sys.stdout.write("Registering flow "
-                                                     "namespace.%s.objects.%s."
-                                                     "flow.%s" %
-                                                     (self.ns_name,obj_name,
-                                                      flow_cls[1].__name__)
-                                                     )
-
+                                logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 17]
```diff
@@ -781,40 +507,16 @@ def _register_subclasses_to_ns(self):
...
-                    payload={"message": "Finding flows in namespace.%s.flows" %
-                             self.ns_name
-                             }
-                )
-            )
-        except KeyError:
-            sys.stdout.write("Finding flows in namespace.%s.flows" %
-                             self.ns_name
-                             )
+        logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

File: tendrl/commons/__init__.py [hunk 18]
```diff
@@ -781,40 +507,16 @@ def _register_subclasses_to_ns(self):
...
-                                               "flows.%s" % (self.ns_name,
-                                                             flow_cls[0])
-                                }
-                            )
-                        )
-                    except KeyError:
-                        sys.stdout.write("Registering flow namespace.%s.flows."
-                                         "%s" % (self.ns_name,flow_cls[0])
-                                         )
+                    logger.log("info", NS.get("publisher_id", None),
```
Comment (r0h4n) [inline tendrl/commons/__init__.py]: "debug"

**Pull request merged**